docs/index.md

```diff
@@ -94,9 +94,18 @@ See how to get started with Flox in your language:
 
 ...one paragraph description, can include links within to services concepts, sharing, etc...
 
-## Reproducible package manager for your whole system
+## Reproducible package manager for any system
+
+Use Flox to set up your [default $HOME environment](https://flox.dev/docs/tutorials/default-environment/) with a set of 
```

Comment:
I think people are going to be confused what `default $HOME environment` means, and may dismiss the product because they think it's messing with the `$HOME` variable. I think `default environment` works just as well.

Reply:
Done, thanks!